scripts/make_docs.py

```diff
@@ -57,16 +59,23 @@
 
 ## Available selectors
 
-{%- for key, val in platforms|dictsort %}
-- `{{key}}`
-{% endfor %}
+{%- for key, val in selectors|dictsort %}
+- `{{key}}`{% endfor %}
+
+## Available Platforms
+Specify which platform to build for via the `--platform` argument. If provided, this argument must be formated as `<platform>-<architecture>`
+> Only `osx` machines may specify `osx` as the target platform.
```

Comment:
Not entirely true. This is true when building the `pkg` installers, but not true when building the `sh` installers

Reply:
Is this correct?
![image](https://user-images.githubusercontent.com/48299585/214204531-8ac77289-6f2b-4c67-8939-db3d0069d829.png)

Reply:
Does Intel vs M1 come into play at all here?

Reply:
Should this section be removed from `CONSTRUCT.md` altogether seeing as how it is now in part of the help message?

Reply:
4326412 Removed the comment.
I was considering adding the table, but this PR is not scoped to alter `installer_type` documentation, which would be the more appropriate location. And after reading it, the docs for that are clear enough